src/cases/components/roles-and-access/roles-and-access.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { CaseView } from '@hmcts/ccd-case-ui-toolkit';
+import { CaseRole } from '../../../../api/workAllocation2/interfaces/caseRole';
+
+@Component({
+  selector: 'exui-roles-and-access',
+  templateUrl: './roles-and-access.component.html',
+  styleUrls: ['./roles-and-access.component.scss']
```

Comment:
Minor query - Is this required? Will there be stylings involved?

Reply:
I will remove 😏